views/js/runner/plugins/tools/hider/contentMask.js

```diff
@@ -0,0 +1,46 @@
+/**
+ * ContentMask component
+ */
+define([
+    'lodash',
+    'i18n',
+    'ui/component',
+    'tpl!taoQtiTest/runner/plugins/tools/hider/contentMask',
+], function(_, __, component, contentMaskTpl) {
+    'use strict';
+
+    var defaultConfig = {
+        content: __('Content hidden'),
+    };
+
+    return function contentMaskFactory($container, config) {
+        var api = {};
+        var contentMask = component(api, defaultConfig);
+
+        contentMask
+            .on('init', function() {
+                this.render($container);
+                this.setState('visible', false);
+                this.hide();
+            })
+            .on('toggle', function() {
+                if (this.is('visible')) {
+                    this.setState('visible', false);
```

Comment:
There is already a state related to `show()` and `hide()`:
https://github.com/oat-sa/tao-core/blob/1a1d9f5bde0d144ff8858aa234295cc2ff37ca90/views/js/ui/component.js#L261